[2023-04-08T10:16:50.499+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: io_resto_casa.respond scheduled__2023-04-08T10:00:00+00:00 [queued]>
[2023-04-08T10:16:50.512+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: io_resto_casa.respond scheduled__2023-04-08T10:00:00+00:00 [queued]>
[2023-04-08T10:16:50.513+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-08T10:16:50.514+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 3
[2023-04-08T10:16:50.515+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-08T10:16:50.534+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): respond> on 2023-04-08 10:00:00+00:00
[2023-04-08T10:16:50.545+0000] {standard_task_runner.py:55} INFO - Started process 13714 to run task
[2023-04-08T10:16:50.549+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'io_resto_casa', 'respond', 'scheduled__2023-04-08T10:00:00+00:00', '--job-id', '321', '--raw', '--subdir', 'DAGS_FOLDER/ciao.py', '--cfg-path', '/tmp/tmpymfrpr8h']
[2023-04-08T10:16:50.553+0000] {standard_task_runner.py:83} INFO - Job 321: Subtask respond
[2023-04-08T10:16:50.660+0000] {task_command.py:389} INFO - Running <TaskInstance: io_resto_casa.respond scheduled__2023-04-08T10:00:00+00:00 [running]> on host cfece00ff772
[2023-04-08T10:16:50.766+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=io_resto_casa
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2023-04-08T10:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-08T10:00:00+00:00
[2023-04-08T10:16:50.769+0000] {python.py:177} INFO - Done. Returned value was: Greet Responded Again
[2023-04-08T10:16:50.803+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=io_resto_casa, task_id=respond, execution_date=20230408T100000, start_date=20230408T101650, end_date=20230408T101650
[2023-04-08T10:16:50.843+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-08T10:16:50.867+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
